src/main/webapp/WEB-INF/view/navbar.jsp

```diff
@@ -9,15 +9,16 @@
   <a id="navTitle" href="/">CodeU Chat App - Nemo</a>
   <a href="/conversations">Conversations</a>
   <% if(navBarUsername != null){ %>
-    <a href="/profile/<%=navBarUser %>">
+    <a href="/profile/<%=navBarUsername %>">
       Hello <%= navBarUsername %>!</a>
   <% } else{ %>
     <a href="/login">Login</a>
   <% } %>
   <a href="/about.jsp">About</a>
   <a href="/activityfeed">Activity Feed</a>
+  <a href="/signout" style="float: right"> Sign out </a>
```

Comment:
If you'll take my rename suggestion above, then this should be `/logout` to match. :)